docs/modeling/example-fitting-model-sets.rst

```diff
@@ -1,37 +1,175 @@
-      >>> y = p1(x)
-      >>> yy = np.array([y, y])
-      >>> p2 = models.Polynomial1D(3, n_models=2)
-      >>> pfit = fitting.LinearLSQFitter()
-      >>> new_model = pfit(p2, x, yy)
-      >>> print(new_model)  # doctest: +SKIP
-      Model: Polynomial1D
-      Inputs: 1
-      Outputs: 1
-      Model set size: 2
-      Degree: 3
-      Parameters:
-           c0  c1         c2                 c3
-          --- --- ------------------ -----------------
-          1.0 2.0 -5.86673908219e-16 3.61636197841e-17
-          1.0 2.0 -5.86673908219e-16 3.61636197841e-17
+Astropy model sets let you fit the same (linear) model to lots of independent
+data sets. It solves the linear equations simultaneously, so can avoid looping.
+But getting the data into the right shape can be a bit tricky. 
+
+The time savings could be worth the effort. In the example below, if we change
+the width*height of the data cube to 500*500 it takes 140 ms on a 2015 MacBook Pro
+to fit the models using model sets. Doing the same fit by looping over the 500*500 models 
+takes 1.5 minutes, more than 600 times slower.
+
+In the example below, we create a 3D data cube where the first dimension is a ramp --
+for example as from non-destructive readouts of an IR detector. So each pixel has a 
+depth along a time axis, and flux that results a total number of counts that is 
+increasing with time. We will be fitting a 1D polynomial vs. time to estimate the 
+flux in counts/second (the slope of the fit).
+
+First, import the necessary libraries:
+
+.. doctest-requires:: matplotlib
+
+    >>> import numpy as np
+    >>> import matplotlib.pyplot as plt
+    >>> from scipy import stats
+    >>> from astropy.modeling import models, fitting
+
+We will use just a small image of 3 rows by 4 columns, with a depth of 10 non-destructive reads::
+
+    >>> depth, width, height = 10, 3, 4  # Time is along the depth axis
+    >>> t = np.arange(depth, dtype=np.float64)*10.  # e.g. readouts every 10 seconds
+
+The number of counts in neach pixel is flux*time with the addition of some Gaussian noise::
+
+    >>> fluxes = np.arange(1.*width*height).reshape(height, width)
+    >>> image = fluxes[np.newaxis, :, :] * t[:, np.newaxis, np.newaxis]
+    >>> image += stats.norm.rvs(0., image*0.05, size=image.shape)  # Add noise
+    >>> image.shape
+    (10, 3, 4)
+
+Create the models and the fitter. We need N=width*height instances of the same linear,
+parametric model (model sets currently only work with linear models and fitters)::
+
+    >>> N = width*height 
+    >>> line = models.Polynomial1D(degree=1, n_models=N)
+    >>> fit = fitting.LinearLSQFitter()
+    >>> print("We created %d models" % len(line))
+    We created 12 models
+
+We need to get the data to be fit into the right shape. It's not possible to just feed
+the 3D data cube. In this case, the time axis can be one dimensional. 
+The fluxes have to be organized into an array that is of shape `width*height,depth` --  in 
```

Comment:
@hcferguson because `width*height,depth` i sin single back ticks it attempts to link and fails. Use double back ticks here, 

```
 ``width*height,depth``
```